src/open_prime_hunters_rando/string_tables/string_tables_patches.py

```diff
@@ -26,3 +29,31 @@ def _patch_hints(file_manager: FileManager, hints: dict[str, str]) -> None:
     for string_id, text in hints.items():
         string_entry = scan_log.get_string(string_id)
         string_entry.text = text
+
+
+def _patch_pickups(file_manager: FileManager, game_patches: dict[str, int]) -> None:
+    ammo = game_patches.get("ammo_per_expansion")
+    missiles = game_patches.get("missiles_per_expansion")
```

Comment:
if you use `get`, these can return None.
either make it `get(..., 0)`, or require these fields to be present and use `[]`

Reply:
I don't think it's physically possible to return None here because the default is set in the schema